src/components/PopupWithForm.js

```diff
@@ -26,6 +31,23 @@ export default class PopupWithForm extends Popup {
 
   close() {
```

Comment:
You shouldn’t create the **same methods** of the parent class if you don’t change them inside and only call them with `super`. The parent methods are available in any inheritor of the class.